### ISSUE TITLE
Error when requiring package in Node: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by knordman** [wimpyprogrammer/regex-to-strings#9]
After installing the package, opening Node (v14.15.0), and trying to require the package there is an error about missing regeneratorRuntime. If I pull the .ts code and transpile it as part of my own project it works great. So I guess this has to do with the .js build / babel settings. I assume the package is intended to work in Node also?

```
Welcome to Node.js v14.15.0.
Type ".help" for more information.
> r = require('regex-to-strings')
Uncaught ReferenceError: regeneratorRuntime is not defined
    at toIterable (/...redacted.../node_modules/regex-to-strings/lib/Expansion.js:13:3)
    at new Expansion (/...redacted.../node_modules/regex-to-strings/lib/Expansion.js:52:22)
    at Object.<anonymous> (/...redacted.../node_modules/regex-to-strings/lib/Expansion.js:58:37)
```

**Comment by wimpyprogrammer** [comment]
Hi Kristian, sorry I didn't respond sooner.  I must have a problem with my GitHub notification settings.

Thanks so much for reporting this issue!  I've adjusted the Babel settings and published version [2.0.1](https://github.com/wimpyprogrammer/regex-to-strings/releases/tag/2.0.1).  Please let me know if you still encounter any issues!

**Comment by knordman** [comment]
Thank you very much, I can confirm that everything works as expected with `2.0.1`  :+1:
